include/myst/sharedmem.h

```diff
@@ -53,9 +53,16 @@ void myst_shmem_mremap_update(
 
 bool myst_shmem_can_mprotect(shared_mapping_t* sm, void* addr, size_t length);
 
-bool myst_addr_within_process_owned_shmem(
+/*
+Return value:
+0 - if no shared memory region corresponding to entire range.
```

Comment:
Which return value is the error? The Linux convention is that return values less than zero are errors.